.github/workflows/test.yml

```diff
@@ -62,6 +57,13 @@ jobs:
       - name: Setup Gradle options
         run: echo "GRADLE_OPTIONS=-Porg.gradle.java.installations.auto-detect=false -Porg.gradle.java.installations.fromEnv=JAVA_HOME_11_x64,JAVA_HOME_8_x64" >> $GITHUB_ENV
 
+      - name: Create Javadocs
+        run: |
+          cd ${GITHUB_WORKSPACE}/newrelic-java-agent/newrelic-api 
+          ../gradlew javadoc $GRADLE_OPTIONS -Prelease=true -PagentVersion=8.0.0
```

Comment:
I also added this $GRADLE_OPTIONS which makes sure we are using the Java we installed.